server/server.go

```diff
@@ -48,10 +52,25 @@ func New(c *Config) (*Server, error) {
 		return nil, errors.Wrap(err, "failed to initialize base server")
 	}
 
+	maxSize := int64(50 * bytesize.MB)
+	if c.Cache.MaxSize != "" {
+		maxSizeBytes, err := bytesize.Parse(c.Cache.MaxSize)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to parse max size")
+		}
+		maxSize = int64(maxSizeBytes)
+	}
+
+	maxAge := 20 * time.Minute
+	if c.Cache.MaxAge != 0 {
+		maxAge = c.Cache.MaxAge
+	}
+
 	userAgent := fmt.Sprintf("%s/%s", c.Options.AppName, version.GetVersion())
 	clientCreator, err := githubapp.NewDefaultCachingClientCreator(
 		c.Github,
 		githubapp.WithClientUserAgent(userAgent),
+		githubapp.WithClientCaching(true, func() httpcache.Cache { return lrucache.New(maxSize, maxAge.Nanoseconds()) }),
```

Comment:
From https://godoc.org/github.com/die-net/lrucache#New:

>  If maxAge > 0, entries will also be expired after maxAge seconds.

That said, I'm more interested in the size limit than the time limit... I'm not sure if we get any benefit from time-based expiration as long as it can't use unbounded memory.

Reply:
I'd be fine just removing that param 👍